www/src/routes/MockInterview.tsx

```diff
@@ -0,0 +1,5 @@
+import React, { FC } from 'react';
```

Comment:
Yeah, that's what I was thinking. The other alternative was to create `MockInterviewPublic` and `MockInterviewPrivate` (and the other routes too) pages to completely separate the content. Then conditionally render based on the Auth0 hook in `App`. But I figured that since the public pages are largely just gonna be static text, we can just hold it in the same page: `MockInterview` and render conditionally within that. To keep everything bundled appropriately.

Reply:
Makes sense :+1: